client/tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py3
+
+[testenv]
+deps = -e .[test,queuable,sentry]
+commands = py.test {posargs}
+usedevelop = True
+setenv =
+    PIP_INDEX_URL = https://pip-spacecraft.earth.planet.com/simple
```

Comment:
Do we really want to use this index in the open source datalake?

Reply:
Nope!